lightning/src/chain/keysinterface.rs

```diff
@@ -245,12 +247,24 @@ pub trait ChannelKeys : Send+Clone {
 	/// return value must contain a signature.
 	fn sign_local_commitment_htlc_transactions<T: secp256k1::Signing + secp256k1::Verification>(&self, local_commitment_tx: &LocalCommitmentTransaction, local_csv: u16, secp_ctx: &Secp256k1<T>) -> Result<Vec<Option<Signature>>, ()>;
 
+	/// Signs a justice transaction.
```

Comment:
These need real documentation, a number of the parameter names don't appear to be correct, etc. But I'm sure you knew that :)

Reply:
Yeah I knew it. But was expecting better documentation to be part of your key-interface cleanup but it got merged first ;)